modules/sb-start-asb.adoc

```diff
@@ -19,8 +19,20 @@ using the following procedure.
 // TODO: Change to *OpenShift Ansible Service Broker Operator* by GA (2 of 5)
 . Select the *Automation Broker Operator*.
 . Under *Provided APIs*, click *Create New* for *Automation Broker*.
-. Review the default YAML, set any additional {asb-name} configuration options,
-and click *Create*.
+. Add the following to the `spec` field in the default YAML provided:
++
+[source,yaml]
+----
+registry:
+  - name: rhcc
+    type: rhcc
+    url: registry.access.redhat.com
+    auth_type: secret
+    auth_name: asb-auth-secret
+----
++
```

Comment:
`registry.access.redhat.com` is used as default before v3.10, after v3.11+, the default changed to `registry.redhat.io`

If using `registry.access.redhat.com`, do NOT need secret, like following:
```
      - type: rhcc
        name: rh
        url:  https://registry.access.redhat.com
        org:  
        white_list: [.*-apb$]
        auth_type: ""
        auth_name: ""
```